File: homework_6/calculations/Plus.java
package calculations;

//класс для нахождения суммы двух чисел
public class Plus implements I__Calculator {
    private double a;
    private double b;
  
    public Plus(double a, double b) {
       this.a = a;
       this.b = b;
    }
  
    @Override
    public double getResult() {
       return a + b;
    }
}

File: homework_7/calculations/plus.py
from calculations.I__calculator import Calculations

"""наследуем класс Plus от базового Calculations"""
class Plus(Calculations):
    def __init__(self, a, b):
        self.a = a
        self.b = b
    
    """переопределяем абстрактый метод"""
    def get_result(self):
        return self.a + self.b
